public/app/core/internationalization/index.tsx

```diff
@@ -56,7 +56,7 @@ export function changeLanguage(locale: string) {
 }
 
 export const Trans: typeof I18NextTrans = (props) => {
-  return <I18NextTrans {...props} />;
+  return <I18NextTrans shouldUnescape {...props} />;
```

Comment:
React prevents XSS attacks out of the box ([link](https://legacy.reactjs.org/docs/introducing-jsx.html#jsx-prevents-injection-attacks)), but we can't be sure that internationalized strings will always be used within React, so settings using `escapeValue: false` in `options` on line 31 doesn't seem safe enough.

Overriding `shouldUnescape` in `<Trans />` means we can be sure it will only be rendered in React components.

Reply:
If your intentions is to always set `shouldUnescape` to true you should place if after the `{...props}` spread statement. Otherwise it can be overridden in the passed props.

Reply:
The intention is that it can be overridden in the passed props.